### PR TITLE
Jormun: fix next last previous links

### PR DESCRIPTION
**Pull request by xlqian**
http://jira.canaltp.fr/browse/NAVITIAII-1980

### CODE REVIEW

File: source/jormungandr/tests/ptref_tests.py [hunk 1]
```diff
@@ -156,6 +161,8 @@ def test_line(self):
         assert line_group[0]['name'] == 'A group'
         assert line_group[0]['id'] == 'group:A'
 
+        self._test_links(response, 'lines')
```
Comment (antoine-de) [inline source/jormungandr/tests/ptref_tests.py:182]: isn't there a is_valid_routes (lines, ...) where you can add this ?

File: source/jormungandr/tests/ptref_tests.py [hunk 2]
```diff
@@ -38,7 +38,12 @@ class TestPtRef(AbstractTestFixture):
     """
     Test the structure of the ptref response
     """
-
+    @staticmethod
+    def _test_links(response, pt_obj_name):
+        for l in response['links']:
+            if l['type'] in ['next', 'last', 'first']:
```
Comment (pbougue) [inline source/jormungandr/tests/ptref_tests.py]: don't you need to manage 'previous' also? (at least it would be nice to have one test on it)

**Comment by pbougue** [comment]
It would be nice to add tests on consistency of next/previous/first/last (self?) if there are no tests on that, but that is not directly related, so...

File: source/jormungandr/tests/ptref_tests.py (hunk 2 above)
Comment (xlqian) [inline source/jormungandr/tests/ptref_tests.py]: you're right

File: source/jormungandr/tests/ptref_tests.py [hunk 3]
```diff
@@ -221,6 +245,8 @@ def test_route(self):
         assert com[0]['type'] == 'standard'
         assert com[0]['value'] == "I'm a happy comment"
 
+        self._test_links(response, 'routes')
```
Comment (pbougue) [inline source/jormungandr/tests/ptref_tests.py:248]: would have been nice to check on the second page of an API to see if previous is OK.

Comment (xlqian) [inline source/jormungandr/tests/ptref_tests.py:248]: too late :smiling_imp:

Comment (pbougue) [inline source/jormungandr/tests/ptref_tests.py:248]: You can still take care of my general remark on links consistency in a dedicated PR  :wink:

**Pull request merged**